code/datums/jobs.dm

```diff
@@ -395,6 +395,40 @@
 		return
 #endif
 
+/datum/job/command/head_of_catering
```

Comment:
Be sure to implement gib-rifling for this job.

Reply:
Done, will be in next push